content/go/concepts/loops/loops.md

```diff
@@ -0,0 +1,149 @@
+---
+Title: 'Loops'
+Description: 'There are several different types of loops in Go. All are written using the `for` statement with different syntaxes.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Control Flow'
+  - 'Iterators'
+  - 'Loops'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/back-end-engineer-career-path'
+---
+
+There are several different types of loops in Go. All are written using the `for` statement with different syntaxes.
+
+## Three-Component Loop
+
+The three-component loop syntax is the same as those used in C or Java. This is often called a `for` loop in other languages.
```

Comment:
```suggestion
In Go, loops are defined with a three-component syntax similar to what is used in `for` loops for other languages like C or Java.
```

Reply:
Should something also be said about how Go does not use parenthesis for writing `for` loops?